modules/ROOT/pages/index.adoc

```diff
@@ -16,65 +16,32 @@ Some of the capabilities of Anypoint Runtime Fabric include:
 [NOTE]
 Contact your Customer Success Manager for information on how enable Anypoint Runtime Fabric on your account.
 
-== Runtime Fabric and Other PaaS Providers
+There are two management options for Anypoint Runtime Fabric:
 
-Anypoint Runtime Fabric contains all of the components it requires. These components, including Docker and Kubernetes, are optimized to work efficiently with Mule runtimes and other MuleSoft services.
+* *Runtime Fabric on self-managed Kubernetes*: a version of Runtime Fabric that you install on an existing Kubernetes environment that you operate and manage. This version supports Amazon Elastic Kubernetes Service (EKS) and Azure Kubernetes Service (AKS)
 
-[WARNING]
-Installing Runtime Fabric within an existing Kubernetes-based PaaS is not supported.
+* *Runtime Fabric on VMs / bare metal*: a version of Runtime Fabric where MuleSoft provides the required software components, including Docker and Kubernetes. You install this version on virtual machines that you operate and manage.
 
-If you are already using a PaaS solution, MuleSoft recommends deploying Runtime Fabric in parallel with your PaaS. This enables you to take advantage of the complete benefits of Anypoint Platform.
+Each of these options has its own system requirements and installation procedures. See xref:index-self-managed.adoc[Runtime Fabric Self-managed] and xref:index-appliance.adoc[Runtime Fabric Appliance] for more information.
 
-== Shared Responsibility for On-Premises Runtime Fabric
+== Feature Support Matrix for the Self-managed and Appliance Versions of Runtime Fabric
 
-The successful operation of on-premises Runtime Fabric instances is a shared responsibility. It is critical to understand which areas you must manage and which areas are managed by MuleSoft. Runtime Fabric operates successfully only if it is provided the required infrastructure and maintenance.
+[%header,cols="3*a"]
+.Supported features for the self-managed and appliance versions of Runtime Fabric
+|===
+| | RTF Self-managed | RTF Appliance
+| Support for deploying Mules and API Gateways | Supported | Supported
```

Comment:
Looking at this table, I think we can remove the "Support for" prefix for each of the features -- seems redundant

Reply:
Done